packages/sdk-themes/src/themes/monite.ts

```diff
@@ -0,0 +1,203 @@
+import { grey } from '@mui/material/colors';
+import { Components } from '@mui/material/styles/components';
+import { Palette, PaletteOptions } from '@mui/material/styles/createPalette';
+import { Theme, ThemeOptions } from '@mui/material/styles/createTheme';
+import { TypographyOptions } from '@mui/material/styles/createTypography';
+import type {} from '@mui/x-data-grid/themeAugmentation';
```

Comment:
Is it correct?

Reply:
yes https://mui.com/material-ui/about-the-lab/#typescript

Reply:
I worry that this will also make a change in the customers' Types? Will this augmentation only affect _our theme_ types, or will customer types be augmented as well?

Reply:
since `@mui/x-data-grid/themeAugmentation` uses `declare module`, not `declare global`, I suppose it should affect only the current theme.